server/backend/courses/views.py

```
@@ -16,22 +17,56 @@ class CourseDetailView(generics.RetrieveAPIView):  # For retrieving a specific c
-    serializer = CourseContentsSerializer(content, data=request.data, partial=True)
+#     if serializer.is_valid():
+#         serializer.save()
+#         return Response(serializer.data, status=status.HTTP_200_OK)
     
-    if serializer.is_valid():
-        serializer.save()
-        return Response(serializer.data, status=status.HTTP_200_OK)
+#     return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+
+class AddCourseView(APIView):
+    permission_classes = [IsAuthenticated]
```

Ruff (0.8.2)

Redefinition of unused `permission_classes` from line 4

(F811)